engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/details/subscription-details.controller.js

```diff
@@ -51,9 +51,9 @@ angular.module('Bastion.subscriptions').controller('SubscriptionDetailsControlle
 
             if (limits.length > 0) {
                 return limits.join(", ");
-            } else {
-                return "";
             }
+
+            return "";
```

Comment:
Out of curiosity does eslint recommend this style?  I prefer this style myself:

```javascript
var response = "";

if (limits.length > 0) {
    response = limits.join(", ");
}

return response;
```

Because I find multiple return points difficult to follow in all but the non-trivial example.